resteasy-spring-boot-starter/pom.xml

```diff
@@ -80,19 +87,17 @@
                     <groupId>org.jboss.resteasy</groupId>
                     <artifactId>resteasy-jettison-provider</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>org.jboss.spec.javax.annotation</groupId>
+                    <artifactId>jboss-annotations-api_1.2_spec</artifactId>
+                </exclusion>
             </exclusions>
         </dependency>
         <dependency>
             <groupId>org.jboss.resteasy</groupId>
             <artifactId>resteasy-jackson2-provider</artifactId>
             <version>${resteasy.version}</version>
         </dependency>
-        <dependency>
```

Comment:
This can't be removed because it is needed for compilation time in in class [ResteasyApplicationBuilder.java](https://github.com/paypal/resteasy-spring-boot/blob/master/resteasy-spring-boot-starter/src/main/java/com/paypal/springboot/resteasy/ResteasyApplicationBuilder.java#L5-L6). As good practice, you should always have explicitly set in your pom files all dependencies you need for compilation time, instead of relying on eventual transitive dependencies.